src/js/media-container.js

```diff
@@ -248,6 +248,17 @@ function getBreakpoints(breakpoints, rect) {
  * @cssprop --media-slot-display - `display` of the media slot (default none for [audio] usage).
  */
 class MediaContainer extends globalThis.HTMLElement {
+  static get observedAttributes() {
+    return [Attributes.AUTOHIDE, Attributes.GESTURES_DISABLED]
+      .concat(MEDIA_UI_ATTRIBUTE_NAMES)
+      // Filter out specific / complex data media UI attributes
+      // that shouldn't be propagated to this state receiver element.
```

Comment:
Does it break somehow? Is the list wrong if these aren't actually attributes?

Reply:
these are filtered out because they can get pretty big.
but no, it would not break anything.
they just take a lot of HTML space so to speak.

Reply:
only the attribute `MediaUIAttributes.MEDIA_RENDITION_LIST` is filtered out now.
it's set directly via a JS property w/ fallback to attribute